docs/extensions-plugins/cli.mdx

```diff
@@ -0,0 +1,395 @@
+
+```bash
+pieces edit
+```
+This is used to edit both the classification and name of an asset.
+
+
+```bash
+pieces edit --name "new name"
+```
+to edit the name
+
+```bash
+pieces edit --classification python
+```
+to edit the classification
+
+
+You can you -n or -c to change the name and classification respectively with the edit command.
+
+##### Delete an asset:
+
+This will delete an opened asset by using list or search first. If you do not have an opened asset it will open your most recent asset and ask if you'd like to delete it. 
+
+```bash
+pieces delete
+```
+##### clear:
+
+this will clear the terminal and you will stay in the cmd loop and give you clean terminal interface. 
+
+```bash
+pieces clear
+```
+
+#### Search and Query
+##### Perform a fuzzy search:
+
+```bash
+pieces search [your query]
+```
+
+Finds strings that approximately match patterns. Normal search.
+
+##### Perform a Neural Code Search:
+```bash
+pieces search query --mode ncs
+```
+
+Uses machine learning, deep neural networks, and natural language processing. It can understand the intent of a user's search query and match it with the most relevant results.
+
+##### Perform a Full Text Search:
+```bash
+pieces search query --mode fts
+```
+
+Examines all words in a document to find matches to search criteria.
+
+##### change the llm model you are using:
+
+Change the model in the ask command.
+
+```bash
+pieces change_model [MODEL_INDEX]
+```
+##### Conversations command
+###### Coversations
+
+List all conversations that you have. The green conversation is the one that is currently being used in the ask command
+
+```bash
+pieces conversations
+```
+
+###### Conversation
+
+List the messages in the currently using conversation in the ask command. 
+
+```bash
+pieces conversation
+```
+
+You can create a new conversation that will be used in the ask command.
+
+```bash
+pieces conversation -n
+```
+
+Rename a conversation
+
+```bash
+pieces conversation -r My awsome name
+```
+
+Or you can make the model rename it for you
+
+```bash
+pieces conversation -r
+```
+
+Delete a conversation
+
+```bash
+pieces conversation -d
+```
+
+
+
+You can switch the conversation and list its messages. Check the conversations command to get the index
+
+```bash
+pieces conversation x
+```
+
+
+##### Ask a question to a model:
+**Requires quotes around question**
+
+Ask the Pieces Copilot a question. You can add a relevant file, you can also add relevant snippets based on their index shown in the assets list command. In order to change models from the default (GPT3.5), use the `change_model` command You can add a relevance file or snippet index check the `list assets` command.
+You can use `.` or `/` to refer to the current directory 
+
+```bash
+pieces ask "your question" -f /file1 /file2 folder -s 1 2 3
+```
+
+
+
+
+##### Commiting to github
+
+Auto commit the code to github and generate a commit message you can use the `-p` or `--push` flags to push the code to the repo too.
```

Comment:
I think this description should be updated. We should also add some additional details on how it works, ie.e looking at the git diff and then using that to give you a good looking commit message